modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -10,6 +10,7 @@ any questions about the update.
 
 - [Codemod](#codemod)
 - [Component Deprecations](#component-deprecations)
+- [Buttons](#buttons)
 - [ActionBar Component Updates](#actionbar-component-updates)
```

Comment:
- Add info about `SystemIcon, Graphic, Accent` removing the iconref
- Add info about segmented control
- Add info about icon position changing